src/sharding/Shard.js

```diff
@@ -22,7 +23,7 @@ class Shard extends EventEmitter {
     super();
 
     if (manager.mode === 'process') childProcess = require('child_process');
-    else if (manager.mode === 'worker') Worker = require('worker_threads').Worker;
+    else if (manager.mode === 'worker') Worker = workerThreads.Worker;
```

Comment:
Why isn't this lazy loaded, like childprocess?

Reply:
Probably a left-over from webpack support